### PR TITLE
refactor(close-button): added nav-bar back button.

### PR DESCRIPTION
**Pull request by murosfc**
added nav-bar back button to the pages where the menu is not displayed.

I didn't like the results, I also think that those pages you'd better close the page instead of going back. If you are changing password and you regret because you recall the password, and you go back you will be redirected to the page where you input the verification code, if you back again then you will see the page where you input your phone number, does not look safe...

### CODE REVIEW

File: src/app/authentication/change-password-page/change-password-page.component.html [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
   <h3>Sua senha foi atualizada com sucesso!</h3>
   <br />
   <button appButton (click)="returnToLogin()">Fazer login</button>
   <br />
   <p>Você será redirecionado automaticamente em 3 segundos.</p>
 } @else {
-  <h3>Defina sua nova senha</h3>
-  <p>Preecha os campos abaixo para configurar uma nova senha para sua conta</p>
+  <h4>Defina sua nova senha</h4>
+  <p>Preencha os campos abaixo para configurar uma nova senha para sua conta</p>
```
Comment (rosostolato) [inline src/app/authentication/change-password-page/change-password-page.component.html:11]: **praise**: good catch!

File: src/app/authentication/change-password-page/change-password-page.component.ts [hunk 2]
```diff
@@ -29,11 +29,12 @@ type FormControlGroup<T> = FormGroup<{
   selector: 'app-change-password',
   standalone: true,
   imports: [
+    ButtonDirective,
```
Comment (rosostolato) [inline src/app/authentication/change-password-page/change-password-page.component.ts:32]: **praise**: good usage of the code patterns. Thanks!

**Review by rosostolato — CHANGES_REQUESTED** [review]
I agree with you about the back button. Let's add an input prop to the `Navbar` component to allow us to inform the `href` of the back button. Then we can make it go to home, for example, by passing `/`, or login by passing `/login`.

**Comment by murosfc** [comment]
Changed as suggested. Now we can pass the route and also ask for logout. the default route remains unchanged (go back).

**Review by rosostolato — APPROVED** [review]

**Pull request merged**